packages/nc-gui-v2/components/dashboard/settings/Metadata.vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import { useToast } from 'vue-toastification'
+import { h, useNuxtApp, useProject } from '#imports'
+import MdiReload from '~icons/mdi/reload'
+import MdiDatabaseSync from '~icons/mdi/database-sync'
+
+const { $api } = useNuxtApp()
+const { project } = useProject()
+const toast = useToast()
+
+let isLoading = $ref(false)
+let isDifferent = $ref(false)
+let metadiff = $ref<Array<any>>([])
```

Comment:
Please use the `any[]` notation instead of `Array<any>` 😄 
Both are valid but usually we use `[]` to type annotate arrays, especially to avoid confusion between the `Array` constructor of JS and the Type `Array`